README.md

```diff
@@ -225,12 +225,12 @@ dos-azul-lambda can be configured by setting a number of environment variables:
 * Set `DATA_BDL_DOCTYPE` to override the name of the ElasticSearch document type
   that dos-azul-lambda should expect to correspond with `DATA_BDL_INDEX`. By
   default, this is `databundle`.
-* Set `ES_HOST` to override the hostname of the ElasticSearch instance. On dev,
-  this points to the `dss-azul-commons` ElasticSearch instance
-  (`search-dss-azul-commons-lx3ltgewjw5wiw2yrxftoqr7jy.us-west-2.es.amazonaws.com`).
-  This variable should be specified without the protocol (i.e. without a leading
-  `http://` or `https://`). dos-azul-lambda will not run unless this variable is
-  specified.
+* Set `ES_HOST` to specify the hostname of the ElasticSearch instance. **This
+  must be manually set.** The endpoint should be specified without a leading
+  protocol (e.g. `search-es-instance-12345.us-west-2.es.amazonaws.com`).
+  For the `dev` stage deployment of dos-azul-lambda, `ES_HOST` points to
```

Comment:
I know I'm nitpicking this paragraph to death, but looking at the config.json, which I wasn't aware of until you pointed it out here in  the next sentence, but all the stages point to dos-azul-lambda, don't they?

Reply:
Yes, though I don't know if that's the correct configuration - I've been advised that `staging` will be changed in the near future, and I'm not sure what `prod` is configured with. I think you're right that that should be changed, though!